runner/src/main.rs

```diff
@@ -614,7 +618,7 @@ fn run_example(opt: &RunExamples, example: &Example) -> Step {
     // clients in the foreground.
     #[allow(clippy::collapsible_if)]
     let run_backend_server_clients: Step = if opt.run_server.unwrap_or(true) {
-        let run_server_clients = if opt.run_clients.unwrap_or(true) {
+        let run_server_clients = if opt.client_variant != "none" {
```

Comment:
Could you make `none` a const somewhere?

Reply:
Done